src/main/flight/pid.c

```diff
@@ -86,13 +90,30 @@ void pidResetErrorAngle(void)
 
 void pidResetErrorGyro(void)
 {
-    errorGyroI[ROLL] = 0;
-    errorGyroI[PITCH] = 0;
-    errorGyroI[YAW] = 0;
+    int axis;
+    for (axis = 0; axis < 3; axis++) {
+        errorGyroI[axis] = 0;
+        errorGyroIf[axis] = 0.0f;
+    }
+}
+
+float scaleItermToRcInput(int axis) {
+    float rcCommandReflection = (float)rcCommand[axis] / 500.0f;
```

Comment:
absolute stick reflection with 1 being as full stick input and 0 no stick input.

Is the naming convention confusing here?